test/devapps/sample_cache_accessor.go

```diff
@@ -26,18 +26,18 @@ func (accessor *SampleCacheAccessor) BeforeCacheAccess(cache *storage.Manager) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = cache.Deserialize(data)
+	err = cache.Unmarshal(data)
 	if err != nil {
 		log.Fatal(err)
 	}
 }
 
-func (accessor *SampleCacheAccessor) AfterCacheAccess(cache *storage.Manager) {
-	data, err := cache.Serialize()
+func (accessor *SampleCacheAccessor) ExportCache(cache cache.Marshaler) {
+	data, err := cache.Marshal()
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = ioutil.WriteFile(accessor.file, []byte(data), 0644)
```

Comment:
we should fix this or suppress it

Reply:
I'm assuming you mean the log.Fatal().  I changed these to log.Error.  In reality, there is not much to be done when these happen other than ignore them or log them.  Really up to the developer on what they want to do.

Reply:
Sorry, missing context. gosec linter is failing. If you want to suppress it golangci-lint suppresses with a syntax of

// nolint:gosec

directly above the offending line.

otherwise permissions need to be at 600 or less